recipes/pdf_recipe/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "diffpy.pdfmorph" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 699952c747048db4a383e81fa867887a86480200a56d7a51bdd1ad43d25b392d
+
+build:
+  skip: True  # [py<3]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - matplotlib
+    - diffpy.utils
+    - scipy
+    - numpy
+    - python
+
+test:
+  imports:
+    - diffpy.pdfmorph
+
+about:
+  home: http://github.com/diffpy/diffpy.pdfmorph
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'More quantitative measurements and analyses for spectra taken at different temperatures.'
+
+  description: |
+    PDFmorph is a Python software package designed to increase the insight
+    researchers can get into their spectra collected for atomic pair 
+    distribution functions
+  # doc_url: doc_url to be released in a later version.
+  dev_url: https://github.com/diffpy/diffpy.pdfmorph
+
+extra:
+  recipe-maintainers:
```

Comment:
@chiahaoliu @pavoljuhas @sbillinge are you interested in being maintainers?

Reply:
Yes, it is OK with me.

Reply:
I can contribute too.